modules/reindex/src/internalClusterTest/java/org/opensearch/index/codec/MultiCodecReindexIT.java

```diff
@@ -142,46 +160,7 @@ private void useCodec(String index, String codec) throws ExecutionException, Int
         assertAcked(client().admin().indices().prepareOpen(index).setWaitForActiveShards(1));
     }
 
-    private void flushAndRefreshIndex(String index) {
-
-        // Request is not blocked
-        for (String blockSetting : Arrays.asList(
-            SETTING_BLOCKS_READ,
-            SETTING_BLOCKS_WRITE,
-            SETTING_READ_ONLY,
-            SETTING_BLOCKS_METADATA,
-            SETTING_READ_ONLY_ALLOW_DELETE
-        )) {
```

Comment:
I'm not sure why we wanted to apply index block before executing `flush` and `refresh` and that too for all variants, flushing and refresing. Since, it took unnecessary cycles, I've removed it for now for speed 🚀 

Please, let me know if there's some hidden/tribal knowledge behind this.

Reply:
I think we can remove these blocks for now, don't see a strong reason to keep the blocks since we are only managing ingestion.